src/lib/libast/include/sfio.h

```diff
@@ -191,6 +191,7 @@ struct _sffmt_s
 
 /* namespace incursion workarounds -- migrate to the new names */
 #if !_mac_SF_APPEND
+#undef SF_APPEND_
```

Comment:
This is the wrong fix. See the comment I added to the associated issue. The correct fix, that mimics the old behavior, is to do `#ifdef SF_APPEND`.

P.S., Thank you for opening an issue and tying this PR to that issue. Too many people just create a PR.

Reply:
I am one of them.  Haven't always created an issue along with a PR.

Reply:
Applied your review finding's suggestion.
Awaiting further actions depending on the concern documented in the issue #309.